scripts/language_model/train/transformer_lm.py

```diff
@@ -0,0 +1,233 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Language models."""
+__all__ = ['BERTRNN']
+
+from mxnet.gluon import Block, nn
+from mxnet import nd
+
+from gluonnlp.model.utils import _get_rnn_layer
+
+
+class BERTRNN(Block):
+    """BERT based language model. Paper would be available soon.
```

Comment:
could you add a paragraph of description for the architecture? it would make the argument doc easier to follow.